surface-model/surfacemodel_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix surfacemodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surfacemodel/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+surfacemodel: a owl:Ontology ;
+    rdfs:label "ontology definition for surfaceModel"@en ;
+    dcterms:contributor "Sebastian Tuttas (3DM)" ;
+    owl:versionInfo "0.1"^^xsd:float .
```

Comment:
^^xsd:float can be removed

Reply:
fixed

Reply:
it is still there:
<img width="343" alt="image" src="https://github.com/GAIA-X4PLC-AAD/ontology-management-base/assets/70939848/7d512851-ff11-45c1-b6a8-ef932fdab23a">

Reply:
done in ontology file